portfolio/src/main/webapp/about.html

```diff
@@ -7,68 +7,66 @@
+                <a href="contact.html"> <p> Contact </p> </a>
+            </div>
         </div>
-    </div>
-    <div id="about">
         <h3>Want to get to know me? Let's play Two Truths One Lie</h3>
-        <div id="truth_and_lies_game">
-            <label id="question_num">Press Start to Begin!</label>
+        <div id="truth-and-lies-game">
+            <label id="game-status">Press Start to Begin!</label>
             <div id="questions">
 
                 <!-- These fields will be populated by the about.js file -->
 
-                <div class="truth_and_lies_question">
+                <div class="truth-and-lies-question">
                     <label>
-                        <input type="radio" name="chosen_answer" id="first_answer" value="0">
-                        <img src="/images/two_truth_one_lie_imgs/stars.jpg" id="first_img">
+                        <input type="radio" name="selected-answer" id="first-claim-selected" value="0">
+                        <img src="/images/two_truth_one_lie_imgs/stars.jpg" id="first-claim-image">
                     </label>
-                    <p id="first_txt"></p>
+                    <p id="first-claim-text"></p>
                 </div>
-                <div class="truth_and_lies_question">
+                <div class="truth-and-lies-question">
                     <label>
-                        <input type="radio" name="chosen_answer" id="second_answer" value="1"/>
-                        <img src="/images/two_truth_one_lie_imgs/stars.jpg" id="second_img">
+                        <input type="radio" name="selected-answer" id="second-claim-selected" value="1"/>
+                        <img src="/images/two_truth_one_lie_imgs/stars.jpg" id="second-claim-image">
                     </label>
-                    <p id="second_txt"></p>
+                    <p id="second-claim-text"></p>
                 </div>
-                <div class="truth_and_lies_question">
+                <div class="truth-and-lies-question">
                     <label>
-                        <input type="radio" name="chosen_answer" id="third_answer" value="2"/>
-                        <img src="/images/two_truth_one_lie_imgs/stars.jpg" id="third_img">
+                        <input type="radio" name="selected-answer" id="third-claim-selected" value="2"/>
+                        <img src="/images/two_truth_one_lie_imgs/stars.jpg" id="third-claim-image">
                     </label>
-                    <p id="third_txt"></p>
+                    <p id="third-claim-text"></p>
                 </div>
             </div>
-            <button id="two_truths" type="submit" onClick=next()>Start</button>
-            <button id="tutorial" onclick="open_tutorial()">How to Play</button>
+            <button id="advance-game-btn" type="submit" onClick=next()>Start</button>
+            <button id="tutorial-button" onclick="openTutorial()">How to Play</button>
```

Comment:
Be consistent in your ID naming... referring to btn vs button.  I would recommend going with "button" for both since it's generally more pleasant to read non-abbreviations =)  But either way, choosing one or the other and being consistent makes things clearer.

Reply:
Changed to advance game button